.kokoro/build.sh

```diff
@@ -32,6 +34,10 @@ gcloud config set project $GCLOUD_PROJECT
 
 # Start functions emulator, if appropriate
 if [[ $PROJECT == functions/* ]]; then
+  export FUNCTIONS_LOG_PATH=$(pwd)/logs/cloud-functions-emulator.log
+  npm install -g @google-cloud/functions-emulator
+  touch "$FUNCTIONS_LOG_PATH"
```

Comment:
Is `touch` right here or do you want to delete the file if it's there already?

Reply:
I want to make sure that the file exists.